frontend/src/pages/Skills/SkillGroup/SkillItem/SkillItem.tsx

```diff
@@ -3,21 +3,28 @@ import { FC } from 'react';
 import Avatar from 'components/Avatar';
 import Tooltip from 'components/Tooltip';
 import { Size } from 'enums/sizes';
+import { Numbers } from 'enums/numbers';
 import { ISkillItemProps } from 'pages/Skills/types';
 
 import { SkillContainer, ImageWrapper, SkillNameWrapper, SkillName } from './styled';
 
-const SkillItem: FC<ISkillItemProps> = ({ name, skillImage }) => (
-  <SkillContainer>
-    <ImageWrapper>
-      <Avatar size={Size.submedium} avatar={skillImage} />
-    </ImageWrapper>
-    <SkillNameWrapper>
-      <Tooltip title={name}>
-        <SkillName>{name}</SkillName>
-      </Tooltip>
-    </SkillNameWrapper>
-  </SkillContainer>
-);
+const SkillItem: FC<ISkillItemProps> = ({ name, skillImage }) => {
+  const NAME_MAX_LENGTH = 25;
```

Comment:
should be in the constants folder or at the top of this file, not inside the component